.cirrus.yml

```diff
@@ -13,7 +13,9 @@ task:
   activate_script: pub global activate flutter_plugin_tools
   matrix:
     - name: publishable
-      script: ./script/check_publish.sh
+      script:
+        - flutter channel stable
```

Comment:
I think it makes more sense to alter `check_publish` to bail if it detects that it's being run on the master channel, or skip the script here if it's possible for us to check it. This will run the test twice and also add in an extra step where we downgrade flutter and re-download all its dependencies. Could be missing something but I don't think there's useful information out of running this twice on the same channel.

Reply:
Never mind, I can't read whitespace indentation. :) The MATRIX below doesn't apply to this script.